server/models/donor.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
```

Comment:
Question:
When do we use es6 `Import` vs `const = require();`?

Reply:
In our case, `import` in the react project and `require()` in the rest of it.

Reply:
:+1: I think i understand. lol

Reply:
`import` isn't implemented in node.js yet and we are able to use it in react, thanks to webpack + babel-plugins.